bucket/neovim-gtk.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/daa84/neovim-gtk",
+    "version": "0.1.1_545",
+    "description": "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.",
+    "license": {
+        "identifier": "GPL3",
```

```suggestion
        "identifier": "GPL-3.0-only",
```
Per https://spdx.org/licenses/